app/assets/stylesheets/article-form-needed-legacy.scss

```diff
@@ -1,10 +1,6 @@
 @import 'variables';
 @import 'mixins';
 
-.hidden {
```

Comment:
No need for this one, since we already have styling for that from utility classes.